src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31122,99 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033029 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033029 "2023-04-04T10:53:33Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033029 "DB3a cell")
+AnnotationAssertion(rdfs:label obo:CL_4033029 "diffuse bipolar 3a cell")
+SubClassOf(obo:CL_4033029 obo:CL_0000750)
+
+# Class: obo:CL_4033030 (diffuse bipolar 3b cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033030 "An OFF calbindin-negative diffuse bipolar cell that has a large dendritic field and stratifies narrowly close to the middle of the Inner plexiform layer. Its axon terminal is characterized by regularly branching and varicose processes resembling beads on a string.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033030 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033030 "2023-04-04T10:53:42Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033030 "DB3b cell")
+AnnotationAssertion(rdfs:label obo:CL_4033030 "diffuse bipolar 3b cell")
+SubClassOf(obo:CL_4033030 obo:CL_0000750)
+
+# Class: obo:CL_4033031 (diffuse bipolar 4 cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27833534") Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033031 "An ON diffuse bipolar cell that has preferential connectivity with ON parasol cells and bistratified amacrine cells.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033031 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033031 "2023-04-04T09:51:10Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033031 "DB4 cell")
+AnnotationAssertion(rdfs:label obo:CL_4033031 "diffuse bipolar 4 cell")
+SubClassOf(obo:CL_4033031 obo:CL_0000749)
+
+# Class: obo:CL_4033032 (diffuse bipolar 6 cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033032 "An ON diffuse bipolar cell that has large axon terminals which show little or no overlap. This cell has a preferential connectivity to narrow thorny ganglion cells.")
```

Comment:
Of note, per PMID:32032773, of the ON diffuse bipolar cell types, this type has the largest dendritic field (differentia).

Reply:
Thank you, now the definition is as follows:

"An ON diffuse bipolar cell that has a large dendritic field and large axon terminals, which show little or no overlap. This cell has a predominant connectivity to narrow thorny ganglion cells."